src/components/Accordion.tsx

```diff
@@ -0,0 +1,100 @@
+import { useId } from "@react-aria/utils";
+import { Dispatch, ReactNode, SetStateAction, useEffect, useState } from "react";
+import { useFocusRing } from "react-aria";
+import { Css, Icon, Palette, useTestIds } from "src/index";
+
+export interface AccordionProps {
+  title: ReactNode;
+  children: ReactNode;
+  disabled?: boolean;
+  defaultExpanded?: boolean;
+  size?: AccordionSize;
+  /** Adds a top border (enabled by default) */
+  topBorder?: boolean;
+  /** Adds a bottom border (disabled by default) */
+  bottomBorder?: boolean;
+  /**
+   * Used by AccordionList
+   * Allows multiple accordions to be expanded simultaneously (enabled by default)
+   */
+  index?: number;
+  setExpandedIndex?: Dispatch<SetStateAction<number | undefined>>;
+}
+
+export function Accordion(props: AccordionProps) {
+  const {
+    title,
+    children,
+    size,
+    disabled = false,
+    defaultExpanded = false,
+    topBorder = true,
+    bottomBorder = false,
+    index,
+    setExpandedIndex,
+  } = props;
+  const testIds = useTestIds(props, "accordion");
+  const id = useId();
+  const [expanded, setExpanded] = useState(defaultExpanded);
+  const { isFocusVisible, focusProps } = useFocusRing();
+
+  useEffect(() => { setExpanded(defaultExpanded); }, [defaultExpanded])
+
+  return (
+    <div
+      {...testIds.container}
+      css={{
+        ...Css.if(topBorder).bt.bGray400.if(bottomBorder && !(!expanded && isFocusVisible)).bb.$,
```

Comment:
**suggestion:** Maybe we should always set `bGray400`. Right now if `topBorder === false`, but `bottomBorder === true`, then you won't get the border because aren't setting `bGray400`. 
Also, I don't think we need the `&& !(!expanded && isFocusVisible)` any longer do we after we updated the focus state to use box-shadows.